src/Configuration.ts

```diff
@@ -232,14 +232,10 @@ function loadConfigurationFromEnv(env: any): Partial<Configuration> {
   return configurationFromEnv
 }
 
-function validatePoetVersion(poetVersion: any) {
-  assert(Array.isArray(poetVersion), 'Field poetVersion must be an Array')
-  assert(poetVersion.length === 4, 'Field poetVersion must have 4 elements')
-  poetVersion.forEach((element: any) =>
-    assert(
-      Number.isInteger(element) && 0 <= element && element < 256,
-      'Each member of poetVersion must be an integer between 0 and 255',
-    ),
+function validatePoetVersion(poetVersion: number) {
+  assert(
+    Number.isInteger(poetVersion) && 0 <= poetVersion && poetVersion < 4294967296,
```

Comment:
Shouldn't this be 65536? (256 ** 2)

Reply:
I played around with writeUInt16BE and writeUInt16LE and I found that the maximums numbers are:

```
writeUInt16LE 65535 (2 ^ 16)
writeUInt16BE 4294967296 (2 ^ 32)
```

Reply:
I'm confused. How can endianness affect the size?

Reply:
well, I received this error when you are out the range:

writeUInt16BE:

`RangeError [ERR_OUT_OF_RANGE]: The value of "value" is out of range. It must be >= 0 and <= 4294967295. Received 4294967296`

writeUInt16LE:
`RangeError [ERR_OUT_OF_RANGE]: The value of "value" is out of range. It must be >= 0 and <= 65535. Received -1`

Reply:
Raised an issue about this to NodeJS and they already fixed it! :smile: 

https://github.com/nodejs/node/issues/24205

So we should update our code accordingly.

`Number.isInteger(poetVersion) && 0 <= poetVersion && poetVersion <= 0xFFFF`